client/src/components/SubsiteChangeAlert/tests/SubsiteChangeAlert-test.js

```diff
@@ -0,0 +1,44 @@
+/* global jest, describe, it, expect */
+import React from 'react';
+import SubsiteChangeAlert from '../SubsiteChangeAlert';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+describe('SubsiteChangeAlert', () => {
+  describe('handleRevert', () => {
+    it('delegates to the callback with relveant properties', () => {
+      const callbackFn = jest.fn();
+      const alert = shallow(
+        <SubsiteChangeAlert
+          myTabSubsiteID="1"
+          myTabSubsiteName="one"
+          revertCallback={callbackFn}
+        />
+      );
+      alert.instance().handleRevert();
+      expect(callbackFn.mock.calls).toEqual([['1', 'one']]);
+    });
+  });
+  describe('getMessage', () => {
+    it('should show the old subsite name correctly', () => {
+      const alert = shallow(
```

Comment:
rename this since window.alert() is an existing function, so pretty confusing have a variable with the same name